lib/primitives/element.js

```diff
@@ -30,6 +30,36 @@ var Element = createClass({
     this.content = content !== undefined ? content : null;
   },
 
+  /**
+   * Freezes the element to prevent any mutation.
+   * A frozen element will add `parent` property to every child element
+   * to allow traversing up the element tree.
+   *
+   * @memberof Element.prototype
+   */
+  freeze: function() {
+    if (this._meta) {
```

Comment:
Note, since `this.meta` getter has a side effect of creating an ObjectElement if it does not exist. This would lead into an infinite loop of creating meta on the meta of the meta of the meta.... so I am accessing the internal property directly.